installers/rpm-code/dev-build-all.sh

```diff
@@ -29,6 +29,8 @@ codes=(
 
     bluesky
 
+    raydata
+
```

Comment:
Put on the line right after bluesky since they are related/dependencies(?).

Reply:
94010b4 not deps but related.